README.md

```diff
@@ -1,16 +1,71 @@
-<p align="center">
-   <img src="https://github.com/xetdata/pyxet/blob/0c7608c97f6a2a0cb2c83dd38fb717913c4d7522/docs/images/logo.png" alt="logo" width="400" />
-</p>
+# Git-Xet: Scale Git to Terabyte-sized Repos
 
-## What is it?
-xet-core is a collection of Rust crates which provide the low level, high performance, functionality and API bindings for [pyxet](https://pyxet.readthedocs.io/en/latest/) and [git-xet](https://xethub.com/assets/docs/getting-started/install). 
+Git Xet is a plugin for git that allows git repositories to scale to terabytes of data.  
 
-For more details, see [Git is for Data](https://www.cidrdb.org/cidr2023/papers/p43-low.pdf).
+## Design Goals
+
+### Binary Content Deduplication
+
+Binary content is deduplicated across commits, files, and blocks of identical content in otherwise different files.   This allows git to efficiently work with repositories containing large datasets that evolve over time.  See our paper at CIDR 2023, [Git is for Data](https://www.cidrdb.org/cidr2023/papers/p43-low.pdf), for more information on the technical details of this process.  
+
+### Ease of Use
+
+Once a repo is set up to use git-xet, nothing more is required. No per-file configuration or tracking is necessary.
+
+### Scalability
+
+Using the `git xet mount` feature, any branch or commit can be mounted locally as a read-only folder.  The data in this folder is seamlessly materialized as needed, allowing large repositories to be viewed quickly.  This is built using our parallel project, [nfsserve](https://github.com/xetdata/nfsserve).
+
+### Flexibility 
+
+Git-xet naturally integrates seamlessly with [xethub](https://about.xethub.com/?), which provides a number of nice perks, such as data analytics and integration with tools like [xetcache](https://github.com/xetdata/xetcache) and [pyxet](https://pyxet.readthedocs.io/en/latest/) for easy access to your data from any environment.  It also integrates with GitHub or can be used completely localling against a local data store.  
```

Comment:
Maybe add a link to our XetData app documentation (https://xethub.com/assets/docs/github-app#xetdata-github-app) when mentioning GitHub? We have been linking to the documentation instead of the XetData app page itself as it is a better onboarding experience for users to follow the steps on this page.